packages/tabs/src/TabTitle.tsx

```diff
@@ -176,9 +173,15 @@ const listTitleStyles = css`
 `;
 
 const listTitleChildrenStyles = css`
-  display: flex;
-  align-items: center;
-  gap: 4px;
+  width: 100%;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  > svg {
+    vertical-align: bottom;
```

Comment:
Technically `vertical-align: text-bottom;` would be more correct, but this works as well